controller/movies.js

```diff
@@ -0,0 +1,47 @@
+const movies = require('../movies')
+
+const getAllMovies = (request, response) => {
+  return response.send({ movies })
+}
+
+const getMoviesByTitle = (request, response) => {
+  const { title } = request.params
+  const findTitle = movies.filter((movie) => {
+    return movie.title.toLowerCase().includes(String(title))
```

Comment:
change line  10 to includes(title.toLowerCase()) to support upper or lower case search